dialogplus/src/main/java/com/orhanobut/dialogplus/GridHolder.java

```diff
@@ -16,6 +16,7 @@
 
     private static final String TAG = GridHolder.class.getSimpleName();
 
+    private int backgroundColor;
     private final int columnNumber;
```

Comment:
would be nice to put an empty line between final and non-final fields. And I would also move final one to the top